packages/x-data-grid/src/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -241,6 +242,18 @@ function extractColumnWidths(
       }
     }
 
+    if (options.includeHeaderFilters) {
```

Comment:
It should check whether header filters are available, and do nothing if there's no header filters

Reply:
Done